anyway/widgets/suburban_widgets/fatal_accident_yoy_same_month.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict
+
+from flask_babel import _
+
+from anyway.request_params import RequestParams
+from anyway.backend_constants import AccidentSeverity
+from anyway.models import AccidentMarkerView
+from anyway.widgets.widget import register
+from anyway.widgets.suburban_widgets.sub_urban_widget import SubUrbanWidget
+from anyway.widgets.widget_utils import (
+    get_accidents_stats,
+)
+from anyway.models import AccidentMarker
+
+
+@register
+class FatalAccidentYoYSameMonth(SubUrbanWidget):
+    name: str = "fatal_accident_yoy_same_month"
+
+    def __init__(self, request_params: RequestParams):
+        super().__init__(request_params, type(self).name)
+        self.rank = 8
```

Comment:
not sure what rank means and what is the correct number here

Reply:
In the past it was used to order the widgets. It current role is unclear. For now, it suffice to set a number that is different from other widgets.

Reply:
done